Tests/js/views/dashboard/assets/ez-dashboardblockallcontentview-tests.js

```diff
@@ -0,0 +1,245 @@
+
+        setUp: function () {
+            this.view = new Y.eZ.DashboardBlockAllContentView(VIEW_CONFIG);
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+        },
+
+        'Should get data when view get active': function () {
+            var view = this.view,
+                isEventFired = false;
+
+            view.on('getAllContent', function () {
+                isEventFired = true;
+            });
+
+            view.set('active', true);
+
+            Y.Assert.isTrue(isEventFired, 'Should fire the `getAllContent` event');
+        }
+    });
+
+    uiEventsTest = new Y.Test.Case({
+        name: 'eZ Dashboard All Content Block View UI events test',
+
+        setUp: function () {
+            var prepareMock = function (mock, values, callCount) {
+                    Y.Mock.expect(mock, {
+                        method: 'get',
+                        args: [Y.Mock.Value.String],
+                        run: function (key) {
+                            if (values[key]) {
+                                return values[key];
+                            } else {
+                                Y.Assert.fail('Trying to get unexpected attribute value: ' + key);
+                            }
+                        },
+                        callCount: callCount
+                    });
+                },
+                modificationDate = new Date().toString();
+
+            this.contentMock = new Y.Mock();
+            this.versionMock = new Y.Mock();
+            this.contentTypeMock = new Y.Mock();
+
+            prepareMock(this.contentMock, {
+                id: 'alter-bridge',
+                name: 'Alter Bridge',
+                resources: {MainLocation: '/location/1/2/3'},
+                mainLanguageCode: 'eng-GB',
+                currentVersion: this.versionMock
+            }, 5);
+
+            prepareMock(this.versionMock, {
+                versionNo: '2013',
+                modificationDate: modificationDate
+            }, 2);
+
+            prepareMock(this.contentTypeMock, {
+                names: {'eng-GB': 'Rock'}
+            }, 1);
+
+            this.view = new Y.eZ.DashboardBlockAllContentView(VIEW_CONFIG);
+        },
+
+        tearDown: function () {
+            this.view.destroy({remove: true});
+        },
+
+        'Should display row options after clicking on a row': function () {
+            var view = this.view,
+                row;
+
+            view.set('contentList', [{
+                content: this.contentMock,
+                contentType: this.contentTypeMock
+            }]);
+
+            row = view.get('container').one(SELECTOR_ROW);
+
+            row.one('td').simulateGesture('tap', Y.bind(function () {
+                this.resume(Y.bind(function () {
+                    Y.Assert.isTrue(row.hasClass(CLASS_ROW_SELECTED), 'The row should be selected');
+
+                    Y.Mock.verify(this.contentMock);
```

Comment:
those asserts have nothing to do with the row becoming selected so they should be removed.